service/provider/src/main/java/gov/va/vro/service/provider/MasPollingProcessor.java

```diff
@@ -16,27 +21,56 @@ public class MasPollingProcessor implements Processor {
 
   private final CamelEntrance camelEntrance;
   private final MasDelays masDelays;
+  private final MasApiService masService;
 
   @Override
   public void process(Exchange exchange) {
     var payload = exchange.getMessage().getBody(MasClaimDetailsPayload.class);
     log.info("Checking collection status for collection {}.", payload.getCollectionId());
     // call pcCheckCollectionStatus
-    boolean ready = checkCollectionStatus(payload.getCollectionId());
-    if (ready) {
-      log.info("Collection {} is ready for processing.", payload.getCollectionId());
-      // call pcQueryCollectionAnnots
-      // call Lighthouse
-      // Combine results and call PDF generation
-      // if a decision is made, call pcOrderExam
-    } else {
-      log.info("Collection {} is not ready. Requeueing...", payload.getCollectionId());
-      // re-request after some time
-      camelEntrance.notifyAutomatedClaim(payload, masDelays.getMasProcessingSubsequentDelay());
+    try {
+      boolean ready = checkCollectionStatus(payload.getCollectionId());
```

Comment:
As things as now, this will always fail with an exception. We should use some type of mock server to simulate an actual response.

Reply:
I don't think we should create a mock server, since we have to have a real MAS API server to access. If we don't have MAS API access, there is no point to have this MAS API service and mock server.